src/createServer.js

```diff
@@ -1,3 +1,80 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+
+const createServer = () => {
+  const correctCases = [
+    'SNAKE',
+    'KEBAB',
+    'CAMEL',
+    'PASCAL',
+    'UPPER',
+  ];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [url, params] = req.url.split('?');
+    const quertParams = new URLSearchParams(params);
+    const caseName = quertParams.get('toCase');
+    const textToConvert = url.slice(1);
+
+    console.log(textToConvert.length < 1);
+    console.log(!caseName);
+    console.log(!correctCases.includes(caseName));
+
+    if (textToConvert.length < 1
+      || !caseName
+      || !correctCases.includes(caseName)) {
```

Comment:
Currently the checks are duplicated (e.g `!correctCases.includes(caseName)` can be tested twice. Instead just check for messages length (see next comment)
```suggestion
```